src/components/OrderableList/OrderableList.spec.tsx

```diff
@@ -0,0 +1,195 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from "react";
+import { mount } from "@cypress/react";
+import { GridNode } from "@react-types/grid";
+import { DragProperties, ItemDragState, OrderableList, OrderableListContainerProps, OrderableListItem } from ".";
+
+const LIST_ID = "[data-test-id=orderable-list]";
+const LIST_ITEM_ID = "[data-test-id=orderable-list-item]";
+const DRAGGABLE_ITEM = "[data-test-id=draggable-item]";
+const INSERTION_INDICATOR = "[data-test-id=insertion-indicator]";
+const VISUAL_INSERTION_INDICATOR = "[data-test-id=visual-insertion-indicator]";
+const IS_DROP_TARGET = "[data-drop-target=true]";
+
+const FOCUSABLE_ITEM = "[data-test-id=focusable-item]";
+const ITEM_HEIGHT = 20;
+
+type TestItem = {
+    text: string;
+};
+
+const testItems: OrderableListItem<TestItem>[] = [
+    { id: "1", alt: "1", text: "Item 1" },
+    { id: "2", alt: "2", text: "Item 2" },
+    { id: "3", alt: "3", text: "Item 3" },
+    { id: "4", alt: "4", text: "Item 4" },
+    { id: "5", alt: "5", text: "Item 5" },
+    { id: "6", alt: "6", text: "Item 6" },
+];
+
+const renderDefaultTestItems = ({ value }: GridNode<OrderableListItem<TestItem>>) => (
+    <div style={{ height: `${ITEM_HEIGHT}px` }}>{value.text}</div>
+);
+
+const renderWithFocusableItems = (_item: GridNode<OrderableListItem<TestItem>>, { isFocusVisible }: DragProperties) => (
+    <div
+        style={{ height: `${ITEM_HEIGHT}px`, display: "flex", justifyContent: "space-around" }}
+        data-focus-visible={isFocusVisible}
+    >
+        <button data-test-id="focusable-item">Button</button>
+        <input data-test-id="focusable-item" value="Input"></input>
+        <textarea data-test-id="focusable-item">Textarea</textarea>
+    </div>
+);
+
+const getCoordinatesByIndex = (index: number, position: "Before" | "After") => {
+    let y = -15;
+    y += (index + 1) * ITEM_HEIGHT;
+    if (position === "After") {
+        y += 10;
+    }
+    return { x: 20, y };
+};
+
+const OrderableListWithDefaultProps = ({
+    renderContent = renderDefaultTestItems,
+    items = testItems,
+    dragDisabled = false,
+    onMove = cy.stub(),
+}: Partial<OrderableListContainerProps<TestItem>>) => (
+    <OrderableList renderContent={renderContent} dragDisabled={dragDisabled} items={items} onMove={onMove} />
+);
+
+describe("OrderableList Component", () => {
+    it("renders correct number of list items", () => {
+        mount(<OrderableListWithDefaultProps />);
+        cy.get(LIST_ID)
+            .find(LIST_ITEM_ID)
+            .should("have.length", testItems.length)
+            .each(($el, index) => {
+                cy.wrap($el).should("have.text", testItems[index].text);
+            });
+    });
+    it("Does not crash if item list is empty", () => {
```

Comment:
Maybe add a new line after each test to make it more structured, wdyt?

Reply:
Yeah that makes sense, fixed :)